src/languages/de.json

```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
```

Comment:
The point is you half-translated most of those things, "Promote zu Admin" is wierd, ban is usually germanized, so "Benutzer bannen" would make more sense.